### DISCUSSION TITLE
Custom Style for Shapes?

### DISCUSSION THREAD

**Discussion by niblettes** [jgraph/drawio#2689] [Q&A]
Hi, 

Drawio provides several pre-defined styles upper right corner.  These are handy, but don't offer the attributes that I want.

Is there a way to add custom styles to this set?

(and Yes I'm away of how to create a custom library, but a styled shape is not a style).

**Comment by tbleiker** [comment]
Is there an update on this? Looking to do the same!

**Comment by alderg** [comment]
See https://www.drawio.com/doc/faq/configure-diagram-editor

**Reply by tbleiker** [reply]
Oh nice, thx for pointing this out!
